sample/SampleAspNetCoreApp/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SampleAspNetCoreApp.Controllers
+{
+	/**
+	 * The values controller contains a post method for testing the 'CaptureBody' config option of the dotnet apm module
+	 */
+	[Route("api/Values")]
+	public class ValuesController : Controller
+	{
+		// GET api/values
+		//[HttpGet]
+		//public async Task<ActionResult<IEnumerable<string>>> Get()
+		//{
+		//	var lastValue = "call to elastic.co: ";
+		//	try
+		//	{
+		//		var httpClient = new HttpClient();
+		//		var elasticRes = await httpClient.GetAsync("https://elastic.co");
+		//		lastValue += elasticRes.StatusCode;
+		//	}
+		//	catch
+		//	{
+		//		lastValue += "failed";
+		//	}
+
+		//	return new[] { "value1", "value2", lastValue };
+		//}
+
+		//// GET api/values/5
+		//[HttpGet("{id}")]
+		//public ActionResult<string> Get(int id) => "value" + id;
+
+		[HttpPost("{id}")]
```

Comment:
Are you sure the `"{id}"` part is needed? I think if it's there then it must be something like `api/Values/1` - so without `/{id}` URLs won't matc., but the id value comes from `[FromForm]string id`, so the `/1` will be ignored. If you remove it then it'll match to `api/Values`, isn't that the intention? The id comes from the body, not from the url.

Reply:
I removed it. You will see it in the next commit/push.

Reply:
Code commited.